templates/partials/filters.html

```diff
@@ -140,6 +140,70 @@
         </ul>
       </span>
 
+      <span id="formsDropdown" class="filter dropdown mm-dropdown" data-label-no-filter="Form" data-filter-label="facilities" ng-show="filterModel.module.id === 'reporting' && filterModel.type === 'analytics'">
+        <span id="formsDropdownContent" class="mm-button" data-toggle="dropdown">
+          <span class="mm-button-icon">
+            <span class="fa fa-file-o"></span>
+          </span>
+          <span class="mm-button-dropdown">
+            <span class="fa fa-chevron-down"></span>
+          </span>
+          <span class="mm-button-text">{{ filterModel.selectedForm || 'Select Form' }}</span>
```

Comment:
Internationalisation... There are more examples of hardcoded English strings but I'll stop pointing them out now.